docs/en/operations/server-configuration-parameters/settings.md

```diff
@@ -32,6 +32,41 @@ Type: UInt32
 
 Default: `120`
 
+## asynchronous_metric_log
+
+Enabled by default on ClickHouse Cloud deployments.
+
+If the setting is not enabled by default on your environment, depending on how ClickHouse was installed, you can follow the instruction below to enable or disable it.
```

Comment:
`nit:` It's not really a setting, it's a table.
```suggestion
If the table is not enabled by default on your environment, depending on how ClickHouse was installed, you can follow the instruction below to enable or disable it.
```

Reply:
Other than this, the rest of it looks reasonable. Thank you for making the changes.

Reply:
But I think it's fine, we are talking about settings in this section (similar to metric_log section).